From 5e2d80cc1fd8b08446f36729652bfa9b5040a732 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 11:48:55 +0100
Subject: [PATCH 37/53] apps: peel: no reason to keep unused/assigned-only
 variables
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

  peel_server.c: In function ‘my_sctpReadInput’:
  peel_server.c:417:8: warning: variable ‘disped’ set but not used [-Wunused-but-set-variable]
  peel_server.c:416:26: warning: variable ‘s_info’ set but not used [-Wunused-but-set-variable]
  peel_server.c: In function ‘poll_fd’:
  peel_server.c:459:10: warning: variable ‘notdone’ set but not used [-Wunused-but-set-variable]

  peel_client.c: In function ‘my_sctpReadInput’:
  peel_client.c:433:11: warning: variable ‘disped’ set but not used [-Wunused-but-set-variable]
  peel_client.c:432:26: warning: variable ‘s_info’ set but not used [-Wunused-but-set-variable]

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/apps/peel_client.c |  6 +-----
 src/apps/peel_server.c | 14 ++++++--------
 2 files changed, 7 insertions(+), 13 deletions(-)

diff --git a/src/apps/peel_client.c b/src/apps/peel_client.c
index 2c3f234..f604349 100644
--- a/src/apps/peel_client.c
+++ b/src/apps/peel_client.c
@@ -429,17 +429,13 @@ my_sctpReadInput(int fd)
 	/* receive some number of datagrams and
 	 * act on them.
 	 */
-	struct sctp_sndrcvinfo *s_info;
-	int sz,i,disped;
+	int sz;
 	struct msghdr msg;
 	struct iovec iov[2];
 	unsigned char from[200];
 	char readBuffer[65535];
 	char controlVector[65535];
   
-	disped = i = 0;
-
-	s_info = NULL;
 	iov[0].iov_base = readBuffer;
 	iov[0].iov_len = sizeof(readBuffer);
 	iov[1].iov_base = NULL;
diff --git a/src/apps/peel_server.c b/src/apps/peel_server.c
index 9b9b065..88545b4 100644
--- a/src/apps/peel_server.c
+++ b/src/apps/peel_server.c
@@ -413,17 +413,13 @@ my_sctpReadInput(int fd,int maxread)
 	/* receive some number of datagrams and
 	 * act on them.
 	 */
-	struct sctp_sndrcvinfo *s_info;
-	int i,disped;
-
 	struct iovec iov[2];
 	unsigned char from[200];
-	disped = i = 0;
 
 	memset(&msg,0,sizeof(msg));
 	memset(controlVector,0,sizeof(controlVector));
 	memset(readBuffer,0,sizeof(readBuffer));
-	s_info = NULL;
+
 	iov[0].iov_base = readBuffer;
 	iov[0].iov_len = maxread;
 	iov[1].iov_base = NULL;
@@ -456,24 +452,26 @@ int
 poll_fd(int fd)
 {
 	int cameup;
-	int max,notdone;
+	int max;
 	fd_set readfds,writefds,exceptfds;
 	struct timeval tv;
+
 	memset(&tv,0,sizeof(tv));
 	FD_ZERO(&readfds);
 	FD_ZERO(&writefds);
 	FD_ZERO(&exceptfds);
 	cameup = 0;
 	max = fd + 1;
-	notdone = 1;
+
 	printf("poll_fd\n");
 	FD_SET(fd,&readfds);
+
 	select(max,&readfds,&writefds,&exceptfds,NULL);
-	notdone = 0;
 	if(FD_ISSET(fd,&readfds)){
 		printf("Read please\n");
 		cameup += my_sctpReadInput(fd,4100);
 	}
+
 	return(cameup);
 }
 
-- 
1.7.11.7

